nft_ingester/src/processors/accounts_processor.rs

```diff
@@ -163,6 +168,9 @@ impl<T: UnprocessedAccountsGetter> AccountsProcessor<T> {
                                 continue;
                             }
                         };
+
+                        debug!("Processor {}, Unprocessed_accounts: {}  {:?}", self.processor_name, unprocessed_accounts.len(), unprocessed_accounts.iter().map(|account| account.id.to_string()).collect::<Vec<_>>().join(", "));
```

Comment:
as discussed internally, please log the values as follows:
```suggestion
                        debug!(processor = %self.processor_name, unprocessed_accounts_len = %unprocessed_accounts.len(), unprocessed_accounts = ?unprocessed_accounts.iter().map(|account| account.id.to_string()).collect::<Vec<_>>().join(", "), "Processor {}, Unprocessed_accounts: {}  {:?}", self.processor_name, unprocessed_accounts.len(), unprocessed_accounts.iter().map(|account| account.id.to_string()).collect::<Vec<_>>().join(", "));
```

feel free to optimize this as to not iterate twice on unprocessed accounts, but logging the values outside of the "message"/"msg" field in the JSON is a must.